### DISCUSSION TITLE
sender doesn't have enough funds to send tx

### DISCUSSION THREAD

**Discussion by stevegee1** [smartcontractkit/full-blockchain-solidity-course-py#1305] [General]
Trying to deploy a persistent ganache on Brownie FundMe. This is the error flagged: sender doesn't have enough funds to send tx. the upfront cost is:

On my local ganache (UI), the accounts [0] has 99.90eth

**Comment by cromewar** [comment]
Hello @stevegee1 Be sure to select the correct network at deploying with the `--network` flag, and also please share more info on this, the code and screenshoots always are helpful

**Reply by stevegee1** [reply]
I actually don't want to bore us with the code snippets because nothing else changed from Patrick's code. The only issue is that after fixing my persistent ganache on Ethereum "Ganache-locale" as it was called in the video. 
Issue: I got an error deploying, using this command in my terminal, "brownie run scripts/deploy.py --networks ganache-locale."

Error.  Sender doesn't have enough funds

**Comment by n4n0b1t3** [comment]
So you are saying, this is something else than what Patrick is explaining in [05:41:18](https://youtu.be/M576WGiDBdQ?t=20498)?

**Reply by stevegee1** [reply]
I actually don't want to bore us with the code snippets because nothing else changed from Patrick's code. The only issue is that after fixing my persistent ganache on Ethereum "Ganache-locale" as it was called in the video. 
Issue: I got an error deploying, using this command in my terminal, "brownie run scripts/deploy.py --networks ganache-locale."

Error.  Sender doesn't have enough funds
